### DISCUSSION TITLE
Outline behind Authelia: wrong user / group assignment

### DISCUSSION THREAD

**Discussion by Mag1cByt3s** [outline/outline#5131] [Self Hosting] [answered]
Hello,

I am running Outline behind Authelia and I recently moved all my Docker Containers to a new host.

However the database and config is the same as before.

The Problem is that I now have two users in Outline with the same name.

One seems to be my old user from authelia which also has a profile picture I set in Outline and the other one is my current user.

The old account has admin privileges and the new one is not shown as an Admin.

Can someone help me to fix this issue?

Thanks!

![members](https://user-images.githubusercontent.com/48777549/228928440-6155b468-8eba-4414-8360-105a3423a0bc.png)

**Comment by tommoor** [comment, marked as answer]
You'll have to fix it in the database directly, find the `user_authentication` associated with the new `user` record and change the `userId` to point to the old one. You can then sign-in as the admin again and suspend the duplicate user.

**Reply by Mag1cByt3s** [reply]
Good idea, I will try that later, thx!

**Reply by Mag1cByt3s** [reply]
Ok I fixed the issue by setting isAdmin to true for the new account and then suspending the old account through outline.

Sadly I was not able to remove the old account from the database because the userid is referenced through a foreign key constraint a lot.
